File: done/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int get_height(void);

int main(void)
{
    int h = get_height();
    int sp = h - 1;
    int st = 1;

    while (st < h + 1)
    {
        for (int i = 0; i < sp; i++)
        {
            printf(" ");
        }
        for (int i = 0; i < st; i++)
        {
            printf("#");
        }

        printf("  ");

        for (int i = 0; i < st; i++)
        {
            printf("#");
        }

        printf("\n");
        sp -= 1;
        st += 1;
    }
}

int get_height(void)
{
    int i;
    do
    {
        i = get_int("Input Height: ");
    }
    while (i < 1 || i > 8);
    return i;
}

File: done/sentimental-mario-more/mario.py
from cs50 import get_int


def main():
    h = get_height()
    sp = h - 1
    st = 1

    while st < h + 1:
        print(" " * sp, end="")
        print("#" * st, end="  ")
        print("#" * st)
        sp -= 1
        st += 1


def get_height():
    while True:
        h = get_int("Height : ")
        if h > 0 and h < 9:
            return h


main()
